client/my-sites/plans-v2/translated-copy.tsx

```diff
@@ -0,0 +1,203 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { OPTIONS_JETPACK_SECURITY, OPTIONS_JETPACK_SECURITY_MONTHLY } from './constants';
+import {
+	PLAN_JETPACK_COMPLETE,
+	PLAN_JETPACK_COMPLETE_MONTHLY,
+	PLAN_JETPACK_SECURITY_DAILY,
+	PLAN_JETPACK_SECURITY_DAILY_MONTHLY,
+	PLAN_JETPACK_SECURITY_REALTIME,
+	PLAN_JETPACK_SECURITY_REALTIME_MONTHLY,
+} from 'lib/plans/constants';
+import {
+	PRODUCT_JETPACK_ANTI_SPAM,
+	PRODUCT_JETPACK_ANTI_SPAM_MONTHLY,
+	PRODUCT_JETPACK_BACKUP,
```

Comment:
I think `PRODUCT_JETPACK_BACKUP` doesn't actually exist. What we use instead for options/generic cards is `OPTIONS_JETPACK_BACKUP` (and `OPTIONS_JETPACK_BACKUP_MONTHLY`). This fix an issue that appears if you change the term filter from annually to monthly.

Reply:
Good call!